internal/provider/features.go

```diff
@@ -372,6 +372,28 @@ func schemaFeatures(supportLegacyTestSuite bool) *pluginsdk.Schema {
 				},
 			},
 		},
+
+		"netapp": {
```

Comment:
Just as a heads up, since we muxed the provider and added the ability to access [plugin-framework](https://github.com/hashicorp/terraform-plugin-framework) functionality, additional changes will be needed here in order to get the provider to execute successfully. We don't have a contributor guide on this yet, I'll pass the details on to @mbfrahry to help you out with this.

Reply:
This seems to be working well in my tests, not sure what is the concern here, @mbfrahry sent me all necessary files and changes needed to implement this feature flag and so far it works as expected.